utils/create-database.js

```diff
@@ -0,0 +1,67 @@
+// require the promise version of mysql2
```

Comment:
Several of these comments are self explanatory and could be removed, they are describing _how_ you are doing something and not _why_, which is much more useful when others are trying to understand your thought process